src/nanocurrency/util.py

```diff
@@ -93,3 +97,19 @@ def is_hex(h):
         return True
     except ValueError:
         return False
+
+def rpc_request(uri, data, timeout=None):
+    data = json.dumps(data)
+    headers = {'Content-type': 'application/json', 'Accept': 'application/json'}
+    try:
+        response = requests.post(   uri, 
+                                    data = data, 
+                                    headers = headers, 
+                                    timeout = timeout )
+
+        if not response.ok:
+            return None
+        resp_dict = json.loads(response.text)
+        return resp_dict
+    except:
```

Comment:
[Don't use bare except](https://realpython.com/the-most-diabolical-python-antipattern/). You will catch *any* exception which may not even be related to the HTTP request.

You may consider using `response.raise_for_status()`, which raises HTTPError if something goes wrong.